ocs_ci/ocs/pgsql.py

```diff
@@ -338,6 +342,39 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def get_node_utilization(self, master=True, worker=True):
```

Comment:
suggestion:
instead of getting both bool args of master and worker, the function can be modified to get one str arg called `node_type` which will get either `master` or `worker`

Reply:
+1